bridge.js

```diff
@@ -994,6 +1000,24 @@ Receiver.prototype = {
 Emitter(Receiver.prototype);
 
 /**
+ * Error object can't be sent via
+ * .postMessage() so we have to
+ * serialize them into an error-like
+ * Object that can be sent.
+ *
+ * @param  {*} err
+ * @return {Object|*}
+ */
+function serializeError(err) {
+  switch (err && err.constructor.name) {
+    case 'Error': return { message: err.message };
+    case 'DOMError': return { message: err.message, name: err.name };
+    case 'DOMException': return { message: err.toString() };
+    default: return err;
```

Comment:
I noticed that there may be other Error types that we are not checking for and could thus throw clone errors at .postMessage(), eg. `ReferenceError`.  Do yo know of anymore, any suggestions on how to cope with this?